client/src/components/Dedalus.jsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
```

Comment:
I'm going to comment in here and it applies to all other Restaurant components 👍